tests/TestHelpers/FixturesTrait.php

```diff
@@ -78,29 +78,18 @@ protected static function getFixturePath(string $fixtureName, string $subPath =
      *   The expected versions. The keys are the file names, without the .json
      *   suffix, and the values are the expected version numbers, or NULL if
      *   the file should not be present.
-     * @param \ArrayAccess $storage
+     * @param \Tuf\Metadata\StorageInterface $storage
      *   The durable storage for the metadata.
-     *
-     * @return void
      */
-    private static function assertMetadataVersions(array $expectedVersions, StorageInterface $storage): void
+    protected static function assertMetadataVersions(array $expectedVersions, StorageInterface $storage): void
     {
         foreach ($expectedVersions as $role => $version) {
-            switch ($role) {
-                case 'root':
-                    $metadata = $storage->getRoot();
-                    break;
-                case 'timestamp':
-                    $metadata = $storage->getTimestamp();
-                    break;
-                case 'snapshot':
-                    $metadata = $storage->getSnapshot();
-                    break;
-                default:
-                    // Any other roles will be 'targets' or delegated targets roles.
-                    $metadata = $storage->getTargets($role);
-                    break;
-            }
+            $metadata = match ($role) {
```

Comment:
This change is to take advantage of the `match` expression's exhaustiveness and strict type checking.